pkg/dom0-ztools/rootfs/bin/eve

```diff
@@ -251,8 +251,11 @@ __EOT__
           echo "Updated memory-monitor configuration"
           ;;
  verbose) # first lets find our piping process
-          for PIPE in $(pgrep cat); do
-             [ "$(readlink /proc/"$PIPE"/fd/0)" = /run/diag.pipe ] && break
+          while [ "$PIPE" = "" ]
```

Comment:
Outer loop won't help you to fix the problem if there is another `cat` running in the system, so that $PIPE will be not empty.

BTW in case of other `cat` running in the system the further code stops it, which is not nice.

Won't it be more safe to do replace the inner loop with something like:

`for PIPE in $(lsof | grep coreutils | grep /run/diag.pipe | awk '{print $1}')`

Reply:
moved to https://github.com/lf-edge/eve/pull/4119#discussion_r1698371163
because I accidentally pushed the commit into this PR as well.